docs/docsite/rst/roadmap/COLLECTIONS_6.rst

```diff
@@ -57,3 +57,10 @@ Ansible 6.x minor releases will occur approximately every three weeks if changes
 
 
 For more information, reach out on a mailing list or a chat channel - see :ref:`communication` for more details.
+
+Planned work
+============
+
+* Remove compatibility code which prevents parallel install of Ansible 6 with Ansible 2.9 or ansible-base 2.10
+* Stop installing files we have no use for.
+* Ship wheels (as ansible-core 2.13 will likely also do)
```

Comment:
Not sure what the correct full term is. Maybe add a note to say who/when you'd use these.

```suggestion
* Ship Pythno wheels (as ansible-core 2.13 will likely also do)
```

Reply:
```suggestion
* Ship Python wheels (as ansible-core 2.13 will likely also do)
```